### PR TITLE
Combine factors dialog

### PR DESCRIPTION
**Pull request by lilyclements**
@africanmathsinitiative/instat this is ready for testing/review

### CODE REVIEW

**Comment by lilyclements** [comment]
Updated Code to newest version, this is re-ready for review

**Comment by stevekogo** [comment]
This looks OK to me. @dannyparsons Can have a final look before merging.

File: instat/dlgCombine.vb [hunk 1]
```diff
@@ -11,88 +11,85 @@
...
+        ucrFactorsReceiver.Selector = ucrSelectorCombineFactors
+        ucrFactorsReceiver.SetMeAsReceiver()
+        ucrFactorsReceiver.bUseFilteredData = False
+        ucrFactorsReceiver.SetIncludedDataTypes({"factor"})
+
+        ' Input Column Name
+        ucrNewColName.SetIsTextBox()
+        ucrNewColName.SetSaveTypeAsColumn()
+        ucrNewColName.SetDataFrameSelector(ucrSelectorCombineFactors.ucrAvailableDataFrames)
+        ucrNewColName.SetAssignToBooleans(bTempAssignToIsPrefix:=True)
```
Comment (dannyparsons) [inline instat/dlgCombine.vb]: This is needed when we produce multiple columns and the textbox contains a prefix. So not needed here for a single column.

File: instat/dlgCombine.vb [hunk 2]
```diff
@@ -11,88 +11,85 @@
...
+        ucrChkDropUnusedLevels.SetValuesCheckedAndUnchecked("TRUE", "FALSE")
+        ucrChkDropUnusedLevels.SetRDefault("FALSE")
+    End Sub
+
+    Private Sub SetDefaults()
+        Dim clsDefaultFunction As New RFunction
+        ucrSelectorCombineFactors.Reset()
+
+        clsDefaultFunction.SetRCommand("interaction")
+        clsDefaultFunction.SetAssignTo(strTemp:="Interact", strTempDataframe:=ucrSelectorCombineFactors.ucrAvailableDataFrames.cboAvailableDataFrames.Text, strTempColumn:="Interact", bAssignToIsPrefix:=True)
```
Comment (dannyparsons) [inline instat/dlgCombine.vb]: The default should be from the save control, not a fixed value. Also the assign to is not a prefix.

**Pull request merged**